native/themes/markdown-styles.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import type { GlobalTheme } from '../types/themes';
+
+import { getStylesForTheme } from '../themes/colors';
+
+const unboundStyles = {
+  link: {
+    color: 'link',
+    textDecorationLine: 'underline',
+    fontFamily: 'Arial',
+    fontSize: 18,
+  },
+  paragraph: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+  },
+  text: {
+    color: 'listForegroundLabel',
+    fontFamily: 'Arial',
+    fontSize: 18,
+  },
+  emojiOnlyText: {
+    fontFamily: 'Arial',
```

Comment:
The point of using an array for `textStyle` was so that we didn't have to repeat lines like `fontFamily: 'Arial'`. We only need this in the `text` style. You can just copy-paste the `unboundStyles` array I gave you on my last review

Reply:
Sorry, forgot to add markdown-styles to files to push